src/display_output.js

```diff
@@ -1,79 +1,71 @@
-function handleConversion(conversionType) {
+function handleConversion(srcFormat, targetFormat) {
   let num = document.getElementById("num").value;
-  let outputFormat = document.getElementById(conversionType).value;
-  let inputNumber;
+  if (!isValid(num, srcFormat)) return displayWrongOutput(srcFormat);
 
-  switch (conversionType) {
-    case "binToDec":
-      inputNumber = "binary";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(binToDec(num), outputFormat);
-      break;
-    case "binToHex":
-      inputNumber = "binary";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(binToHex(num), outputFormat);
-      break;
-    case "binToOct":
-      inputNumber = "binary";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(binToOct(num), outputFormat);
-      break;
-    case "decToBin":
-      inputNumber = "decimal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(decToBin(num), outputFormat);
-      break;
-    case "decToHex":
-      inputNumber = "decimal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(decToHex(num), outputFormat);
-      break;
-    case "decToOct":
-      inputNumber = "decimal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(decToOct(num), outputFormat);
-      break;
-    case "hexToBin":
-      inputNumber = "hexadecimal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(hexToBin(num), outputFormat);
-      break;
-    case "hexToDec":
-      inputNumber = "hexadecimal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(hexToDec(num), outputFormat);
-      break;
-    case "hexToOct":
-      inputNumber = "hexadecimal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(hexToOct(num), outputFormat);
-      break;
-    case "octToBin":
-      inputNumber = "octal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(octToBin(num), outputFormat);
-      break;
-    case "octToDec":
-      inputNumber = "octal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(octToDec(num), outputFormat);
-      break;
-    case "octToHex":
-      inputNumber = "octal";
-      if (!isValid(num, inputNumber)) return displayWrongOutput(inputNumber);
-      displayOutput(octToHex(num), outputFormat);
-      break;
-    default:
-      return alert("whta did you do?");
+  if ((srcFormat == "binary")) {
```

Comment:
```suggestion
  if (srcFormat === "binary") {
```

Reply:
The double parentheses was something that added prettier when i formatted the document, not sure why.
¿What improves using strict equals on this case? I mean, I'm comparing a string with a string and I don't see the data type could induce an error. Btw, I changed as you suggested.
https://github.com/GonzoVM/bin2dec/commit/2a28d4dc0280f4c01b24b0333fff72d71f7d5d15